src/iperf_api.c

```diff
@@ -2585,6 +2700,12 @@ iperf_reset_test(struct iperf_test *test)
 
     test->other_side_has_retransmits = 0;
 
+    test->total_rate_stats_count = 0;
+    test->total_rate_last_interval_index = 0;
+    test->total_rate_traffic_exceeded_limit = 0;
+    for (int i = 0; i < MAX_INTERVAL; i++)
```

Comment:
Could you move the declaration of `int i` outside the `for()` statement?  It's generating a warning/error on some compilers (notably CentOS 7).